server/src/app.py

```diff
@@ -1,14 +1,15 @@
 from flask import Flask, request, jsonify
 from flask_cors import CORS
-from .chatbot import chatbot # Relative import for unit testing
+from .chatbot import chatbot,read_data # Relative import for unit testing
 
 app = Flask(__name__)
 CORS(app)  # Need to enable cors for the frontend to be able to make requests
 
 @app.route('/chat', methods=['POST'])
 def chat():
     user_message = request.json.get('message')
-    bot_response = chatbot(user_message)
+    response_dictionary = read_data()
```

Comment:
chat() gets called every time a request is made. This still creates a dictionary every time a post request is made.